packages/mask/src/components/GuideStep/index.tsx

```diff
@@ -119,6 +119,12 @@ export default function GuideStep({
         if (disabled) return
         const open = +lastStep === step
         setOpen(open)
+
+        if (!open) return
+        const url = new URL(location.href)
+        if (url.pathname === '/home') return
```

Comment:
It should not suppose the users are on Twitter.

Reply:
Only Twitter has user guide currently.

Reply:
```suggestion
        if (location.pathname === '/home') return
```

Reply:
ping

Reply:
change url to location ? any difference ?